docs/onboarding/Onboarding.md

```diff
@@ -0,0 +1,125 @@
+## Onboarding guide
+
+Playbook Dispatcher is a service that simplifies running Ansible Playbooks on connected hosts.
+This guide explains how an application be integrated with it in order to benefit from:
+
+* common API for dispatching playbooks
+* automated tracking of playbook state
+* common model for representing playbook results
+* built-in authorization support
+
+### Dispatching playbooks
+
+Currently, dispatching of playbooks is only possible with the [Internal/Private API](https://github.com/RedHatInsights/playbook-dispatcher/blob/master/schema/private.openapi.yaml), and it will only be available to accept requests within the cluster.
+
+#### Authenticating with the Internal API
+
+In order to send requests to the Internal API, the client is required to provide a service specific pre-shared key under the `Authorization` header of the HTTP request.
+
+For example:
+```javascript
+POST /internal/dispatch HTTP/1.1
+Authorization: PSK <pre-shared key>
+```
+
+The service specific pre-kshared keys are configured via environment variables.
+You can open a PR in this repository to add a dispatcher pre-shared key for your service.
+The environment variable will need to be in the form of `PSK_AUTH_<service id>`, for example, `PSK_AUTH_REMEDIATIONS`.
+Here is a PR that added a PSK for the Tasks app [#217](https://github.com/RedHatInsights/playbook-dispatcher/pull/217).
```

Comment:
This is not correct. What should happen instead is that the app team requests a new key by contacting somebody from the pipeline workstream (insert our slack handle here). A pipeline workstream member will then follow https://gitlab.cee.redhat.com/service/app-interface/-/blob/master/docs/console.redhat.com/app-sops/playbook-dispatcher/onboarding-new-application.md to set them up